hack/e2e.go

```diff
@@ -262,6 +262,8 @@ func TearDown() bool {
 
 // Up brings an e2e cluster up, recreating it if one is already running.
 func Up() bool {
+	// force having batch/v2alpha1 always on for e2e tests
+	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
```

Comment:
Right, that's why I asked. Again with `batch/v2alpha1` I'm blazing the trail, this time for group versions for e2e tests.